examples/render_to_image_widget.rs

```diff
@@ -70,7 +69,7 @@ fn setup(
     egui_user_textures.add_image(image_handle.clone());
     commands.insert_resource(CubePreviewImage(image_handle.clone()));
 
-    let cube_handle = meshes.add(Mesh::from(shape::Cube { size: 4.0 }));
+    let cube_handle = meshes.add(Mesh::from(Cuboid::new(4.0, 4.0, 4.0)));
```

Comment:
```suggestion
    let cube_handle = meshes.add(Cuboid::new(4.0, 4.0, 4.0));
```
I think now we can skip the conversions now